pip/pep425tags.py

```diff
@@ -124,7 +124,54 @@ def get_platform():
         split_ver = release.split('.')
         return 'macosx_{0}_{1}_{2}'.format(split_ver[0], split_ver[1], machine)
     # XXX remove distutils dependency
-    return distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    if result == "linux_x86_64" and sys.maxsize == 2147483647:
```

Comment:
This occurs because `distutils.util.get_platform()` relies on `os.uname()` (and thus POSIX `uname()`) for the architecture of the platform. It does not make any attempt to determine the "platform" of the interpreter itself.